x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/column_data.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { ClassificationEvaluateResponse } from '../../../../common/analytics';
+
+export function getColumnData(
+  confusionMatrixData: ClassificationEvaluateResponse['classification']['multiclass_confusion_matrix']['confusion_matrix']
```

Comment:
This could be `ConfusionMatrix[]` if you adapt the suggestion further up to split the types into multiple interfaces.